docs/src/pages/components/timeline/timeline.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Timeline React component
+components: Timeline, TimelineItem, TimelineSeparator, TimelineDot, TimelineConnector, TimelineContent, TimelineOppositeContent
+---
+
+# Timeline
+
+<p class="description">The timeline displays a list of events in chronological order.</p>
+
+**Note:** This component is not documented in the [Material Design guidelines](https://material.io/), but Material-UI supports it.
+
+## Simple timeline
+
+Simeple timeline showing list of events.
+
+{{"demo": "pages/components/timeline/SimpleTimeline.js"}}
+
+## Right aligned timeline
+
+Right aligned timeline can show the timeline on the right side of the events.
+
+{{"demo": "pages/components/timeline/RightAlinedTimeline.js"}}
+
+## Alternating timeline
+
+The timeline can display the events on alternating sides.
+
+{{"demo": "pages/components/timeline/AlternateTimeline.js"}}
+
+## Color
+
+The `TimelineDot` can appear in different colors.
+
+{{"demo": "pages/components/timeline/ColorsTimeline.js"}}
+
+## Outlined
+
+{{"demo": "pages/components/timeline/OutlinedTimeline.js"}}
+
+## Opposite content
+
+Timeline can contain an element opposite of the content.
+
+{{"demo": "pages/components/timeline/OppositeContentTimeline.js"}}
+
+## Customized timeline icons in timeline
```

Comment:
”Customized timeline icons” ?

Reply:
Will rename this to `Customized Timeline` as these is more in the example than just customizing icons. What do you think?